test/plugins/node-workspace.ts

```diff
@@ -157,7 +189,7 @@ describe('NodeWorkspace plugin', () => {
             ),
           ],
         }),
-        buildMockCandidatePullRequest('node1', 'node', '2.2.2', {
+        buildMockCandidatePullRequest('node1', 'node', '3.3.3', {
```

Comment:
Why do the mock values need to change? It would be good to see the same tests running with the same assertions against the refactored code.

Reply:
The node1 package (`@here/pkgA`) fixture is already at 3.3.3. The lerna updater only updated the dependency versions, so the output still had 3.3.3 as the `@here/pkgA` version. But switching to the PackageJson updater, it actually did change the value from 3.3.3 to 2.2.2 (which was invalid). To keep the test output the same before and after the refactor, I switched this value to be `3.3.3`.